generators/base/generator-base.mjs

```diff
@@ -53,7 +53,7 @@ import {
   getConfigWithDefaults,
 } from '../../jdl/jhipster/index.mjs';
 
-import databaseData from '../sql-constants.mjs';
+import databaseData, { getDatabaseData } from '../sql/support/database-data.mjs';
```

Comment:
Preferable to use an index.mjs as a hub to `/support` packages: will let us all the ability to modify package names, while providing a clean facade

Reply:
Done